ev3dev/optional.py

```diff
@@ -0,0 +1,678 @@
+"""
+A set of optional modules for interfacing with peripherals (screen and sound)
+"""
+
+import sys
+optional_enabled = sys.implementation.name is not 'micropython'
+
```

Comment:
what are the features needed here missing on micropython? Can those be added as a comment?